### PR TITLE
fix(cloud-connect): fix translation errors

### PR DESCRIPTION
**Pull request by varun257**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | yes/no
| Tickets          | Fix #DTRSD-22064,#DTRSD-22070
| License          | BSD 3-Clause

## Description

update translation for status

### :bookmark: TODOs

- [x] TRANS-28894
- [x] TRANS-29373

Signed-off-by: varun257 <varun257@gmail.com>

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report translator-pull#613.0 ■   
* 
  * pull ■

**Comment by ovh-cds** [comment]
CDS Report translator-pull#613.0 ■   
* 
  * pull ■

**Comment by ovh-cds** [comment]
CDS Report translator-push#11523.0 ✘  
* 
  * push ✘

**Comment by ovh-cds** [comment]
CDS Report translator-push#11523.1 ✘  
* 
  * push ✘

**Comment by ovh-cds** [comment]
CDS Report translator-push#11523.2 ✘  
* 
  * push ✘

**Comment by ovh-cds** [comment]
CDS Report translator-push#11523.3 ✘  
* 
  * push ✘

**Review by marie-j — APPROVED** [review]

**Review by JeremyDec — DISMISSED** [review]

**Review by Heiiji — APPROVED** [review]

**Review by antleblanc — APPROVED** [review]

**Review by JeremyDec — APPROVED** [review]

**Review by marie-j — DISMISSED** [review]

**Review by marie-j — APPROVED** [review]

File: packages/manager/modules/cloud-connect/src/details/service-keys/translations/Messages_fr_FR.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "cloud_connect_service_keys_status_active": "Actif",
   "cloud_connect_service_keys_status_cancelled": "Annulé",
   "cloud_connect_service_keys_status_terminated": "Terminé",
-  "cloud_connect_service_keys_status_toCheck": "Vérifier",
+  "cloud_connect_service_keys_status_toCheck": "Indisponible",
```
Comment (JeremyDec) [inline packages/manager/modules/cloud-connect/src/details/service-keys/translations/Messages_fr_FR.json:6]: Is it right ? I mean, "to check" means "Vérifier" in french.

Comment (varun257) [reply packages/manager/modules/cloud-connect/src/details/service-keys/translations/Messages_fr_FR.json:6]: @JeremyDec 
The step `"ToCheck"` is a temporary procedure on backend performed from OVHcloud side by OVHcloud request. It is supposed to be `"Unavailable"` in english, so they have given equivalent word `"Indisponible"`

**Review by JeremyDec — DISMISSED** [review]

**Review by marie-j — APPROVED** [review]

**Review by JeremyDec — APPROVED** [review]

**Review by antleblanc — APPROVED** [review]

**Pull request merged**